libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -564,4 +541,29 @@ private static void recreateAnalyticsPeriodicBackgroundPublishingWorkRequest() {
                 (long) publishPeriodicallyFrequencyHours
         );
     }
+
+    /**
+     * The available Salesforce analytics publishing types.
+     */
+    public enum SalesforceAnalyticsPublishingType {
```

Comment:
Maybe the values could be:
- PublishDisabled
- PublishOnAppBackground
- PublishPeriodically

Reply:
You got it 💫